src/libLookup/Lookup.cpp

```diff
@@ -2006,8 +2006,11 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
     // Store Tx Block to disk
     bytes serializedTxBlock;
     txBlock.Serialize(serializedTxBlock, 0);
-    BlockStorage::GetBlockStorage().PutTxBlock(
-        txBlock.GetHeader().GetBlockNum(), serializedTxBlock);
+    if (!BlockStorage::GetBlockStorage().PutTxBlock(
+            txBlock.GetHeader().GetBlockNum(), serializedTxBlock)) {
+      LOG_GENERAL(WARNING, "BlockStorage::PutTxBlock failed " << txBlock);
```

Comment:
How about move the error message into `PutTxBlock` then we can handle the debug message in one place for easier maintenance. For caller here, it would be simpler to check the function result only.

Reply:
Yes it make sense to let user know who calls this function. But the point what I mentioned is, the responsibility of telling user what's wrong should be the `callee` instead of `caller`. It's no harm that `caller` give extra information like here, but the `callee` should always takes the responsibility to talk enough about what's going on inside itself. Anyway it should not a roadblock for this PR so I suggest to raise another issue for the log refactoring.